plugins/services/src/js/routes/services.js

```diff
@@ -25,191 +25,194 @@ function buildServiceCrumbs(router) {
-                      }
-                    };
-                  },
-                  children: [
-                    {
-                      type: DefaultRoute,
-                      handler: TaskDetailsTab,
-                      isTab: true,
-                      title:'Details',
-                      buildBreadCrumb() {
-                        return {
-                          parentCrumb: '/services/overview/:id/tasks/:taskID',
-                          getCrumbs() { return []; }
-                        };
-                      }
-                    },
-                    {
-                      type: Route,
-                      path: 'files',
-                      title:'Files',
-                      handler: TaskFilesTab,
-                      isTab: true,
-                      hideHeaderNavigation: true,
-                      fileViewerRoutePath: '/services/overview/:id/tasks/:taskID/view/?:filePath?/?:innerPath?/?',
-                      buildBreadCrumb() {
-                        return {
-                          parentCrumb: '/services/overview/:id/tasks/:taskID',
-                          getCrumbs() { return []; }
-                        };
-                      }
-                    },
-                    {
-                      type: Route,
-                      dontScroll: true,
-                      handler: TaskFileViewer,
-                      title:'Logs',
-                      path: 'view/?:filePath?/?:innerPath?/?',
-                      isTab: true,
-                      buildBreadCrumb() {
-                        return {
-                          parentCrumb: '/services/overview/:id/tasks/:taskID',
-                          getCrumbs() { return []; }
-                        };
-                      }
-                    },
-                    {
-                      type: Route,
-                      path: 'volumes',
-                      handler: VolumeTable,
-                      isTab: true,
-                      title:'Volumes',
-                      buildBreadCrumb() {
-                        return {
-                          parentCrumb: '/services/overview/:id/tasks/:taskID',
-                          getCrumbs() { return []; }
-                        };
-                      }
+              {
+                type: Route,
+                path: 'tasks',
+                component({children}) {
+                  return children;
+                },
+                children: [
+                  {
+                    type: Route,
+                    path: ':taskID',
+                    component: TaskDetail,
+                    hideHeaderNavigation: true,
+                    buildBreadCrumb() {
+                      return {
+                        parentCrumb: '/services/overview/:id',
+                        getCrumbs(params, routes) {
+                          return [
+                            <TaskDetailBreadcrumb
```

Comment:
Again here, we might want to consider using context or another approach to pass down params and routes.